contracts/erc721/UserMintableERC721.sol

```diff
@@ -430,6 +464,65 @@ contract UserMintableERC721 is Initializable, ERC721, ERC721Enumerable, ERC721Me
     );
   }
 
+  /**
+   * @dev Checks that provided document is signed by an identity
+   * and validates and checks if the public key used is a valid
+   * SIGNING_KEY
+   * @param documentRoot bytes32 the anchored document root
+   * @param identity address Identity that signed the document
+   * @param signingRoot bytes32 The message that was signed
+   * @param signature bytes The signature
+   * used to contract the property for precise proofs
+   * @param salt bytes32 salt for leaf construction
+   * @param proof bytes32[] proofs for leaf construction
+   */
+
+  function _requireSignedByIdentity(
+    bytes32 documentRoot,
+    address identity,
+    bytes32 signingRoot,
```

Comment:
Where is the merkle proof that the signingRoot is part of the document?

This is very important otherwise I could just add the signature of an older document version
to it.

Reply:
we can not generate a proof for the signingRoot and it is not necessary. We need the signingRoot just for ecrecover. We prove that the signature is part of the document.